flink-tests/src/test/scala/org/apache/flink/api/scala/runtime/TraversableSerializerTest.scala

```diff
@@ -70,6 +70,7 @@ class TraversableSerializerTest {
   }
 
   @Test
+  @Ignore
```

Comment:
You are right, we should still test this. The entire test was messy. There was a `ScalaCollectionSerializerTestInstance` that was never used. I will move this tests to another class.